tests/api/v1/subjects/get.js

```diff
@@ -395,10 +395,25 @@ describe(`api: GET ${path}`, () => {
     });
   });
 
+  it('returns expected fields when requesting tags field', (done) => {
+    api.get(`${path}?fields=isPublished,name,tags`)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        done(err);
+      }
+
+      expect(res.body[ZERO]).to.have.property('tags');
+      expect(res.body[ZERO]).to.have.property('name');
+      expect(res.body[ZERO]).to.have.property('isPublished');
```

Comment:
well, it _can_, but those kinds of tests tend to be in the db layer... this is just testing that the api layer is doing the right thing